components/lsaHJAS68dskjHfslkdj2.yml

```diff
@@ -0,0 +1,25 @@
+name: emotion-response_selector
```

Comment:
i suggest to follow unofficial rules and name components with "_" not "-".
With "-" we name containers.
(it was important rule when we were using Kubernetes, so we just follow it for now to be able to use kuber in future without breaking changes)